.gitignore

```diff
@@ -67,6 +67,10 @@ npm
 /packages/api-specification/src/bootstrap.min.css
 dist/
 
+# Test output
+/packages/api-tests/electron.json
```

Comment:
Could create a directory for this output, to simplify this and give some more context (i.e. if I see an file called `electron.json`, I'm not sure what it contains, but if it's in a directory, like `coverage` or `config`, then I'd have a better idea what it is).